docs/yaml/ambassador/ambassador-no-rbac.yaml

```diff
@@ -30,7 +33,7 @@ spec:
     spec:
       containers:
       - name: ambassador
-        image: quay.io/datawire/ambassador:%version%
```

Comment:
This definitely seems incorrect. 🙂

Reply:
This definitely is. Thanks for catching it